miio/toiletlid.py

```diff
@@ -0,0 +1,147 @@
+import enum
+import logging
+from typing import Any, Dict
+
+import click
+
+from .click_common import command, format_output, EnumType
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+MODEL_TOILETLID_V1 = "tinymu.toiletlid.v1"
+
+AVAILABLE_PROPERTIES_COMMON = ["work_state", "filter_use_flux", "filter_use_time"]
+
+AVAILABLE_PROPERTIES = {MODEL_TOILETLID_V1: AVAILABLE_PROPERTIES_COMMON}
+
+
+class AmbientLightColor(enum.Enum):
+    White = "0"
+    Yellow = "1"
+    Powder = "2"
+    Green = "3"
+    Purple = "4"
+    Blue = "5"
+    Orange = "6"
+    Red = "7"
+
+
+class ToiletlidStatus:
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    @property
+    def work_state(self) -> int:
+        """Device state code"""
+        return self.data["work_state"]
+
+    @property
+    def is_on(self) -> bool:
+        return self.work
+
+    @property
+    def work(self) -> bool:
```

Comment:
Is there a need for this property, considering there's already a more generic `is_on`?